docs/catalogs/arguments.rst

```diff
@@ -12,6 +12,28 @@ Reading input files
 
 Catalog import reads through a list of files and converts them into a hipscatted catalog.
 
+At a minimum, you need to pass the reader arguments that include where to find the input files,
```

Comment:
I like this a lot. I'd suggest moving this above the "Reading input files" section, since it gives an overview of the different kinds of required inputs, and this section is more detail about the input file formats.